### ISSUE TITLE
New feature request if not already present?

### ISSUE THREAD

**Issue by limitless-code** [OpenVoiceOS/ovos_skill_manager#50]
New feature request if not already present?

New options: 

1. To list currently installed skills
2. remove/ uninstall installed skills.

Thank you.

**Comment by ChanceNCounter** [comment]
Indeed, both of those are in the first-party analogue. I'm gonna repurpose this issue to keep track.

Present in MSM, absent in OSM:

- [ ] `osm list`
- [ ] `osm remove`
- [ ] `osm info`
